processor/tailsamplingprocessor/README.md

```diff
@@ -77,7 +77,7 @@ processors:
           {
             name: test-policy-2,
             type: latency,
-            latency: {threshold_ms: 5000}
+            latency: {threshold_ms: 5000, upper_threshold_ms: 10000}
```

Comment:
Should there be more documentation around this? Perhaps at the description for this policy, close to the top of the readme?

Reply:
I've added a commit for this, which also fixes some lint and adds the necessary changelog file.